src/components/EntityStatus/EntityStatus.tsx

```diff
@@ -95,18 +94,21 @@ export function EntityStatus({
                 </span>
             )}
             <span className={b('link', {'with-left-trim': withLeftTrim})}>{renderLink()}</span>
-            {hasClipboardButton && (
-                <ClipboardButton
-                    text={name}
-                    size="s"
-                    className={b('clipboard-button', {
-                        visible: clipboardButtonAlwaysVisible,
-                    })}
-                />
-            )}
-            {additionalControls && (
-                <span className={b('additional-controls ')}>{additionalControls}</span>
-            )}
+            <div className={b('controls-wrapper')}>
+                {hasClipboardButton && (
+                    <ClipboardButton
+                        text={name}
+                        size="xs"
+                        view="normal"
+                        className={b('clipboard-button', {
+                            visible: clipboardButtonAlwaysVisible,
+                        })}
+                    />
+                )}
+                {additionalControls && (
+                    <span className={b('additional-controls')}>{additionalControls}</span>
+                )}
```

Comment:
`DeveloperUIButton` is controlled by `display: none` and `display: inline-block`. It cannot be selected with Tab. Two ways possible:

1. All buttons in the UI should be keyboard accessible, so this `DeveloperUIButton` button should be updated too

2. We don't need to make buttons in table keyboard accessible (for me in tables it doesn't make much sense anyway), so we can stick to previous solution, but update only `ClipboardButton`

Reply:
Lets try the hardest way number 1. I hope I've found a solution.